Problem importing @lit-labs/motion

Not quite sure whether I'm doing something wrong or `@lit-labs/motion` has been packaged incorrectly, but in addition to the already reported issue with missing type definitions (https://github.com/lit/lit/issues/1866), I am having problems importing `@lit-labs/motion` into my code at all.

# Description
I am building a typescript project with webpack, using babel-loader to transform typescript files. An attempt to import `@lit-labs/motion` (version 1.0.0-rc.2) into my project files:

```ts
// @ts-expect-error - No types available for the package
import {flip} from '@lit-labs/motion';
```

results in the following error:

```
Module not found: Error: Can't resolve '@lit-labs/motion' in '[path to project root]/src/client'                                                                                 
resolve '@lit-labs/motion' in '[path to project root]/src/client'           
  Parsed request is a module                                                                                                                                                                              
  using description file: [path to project root]/package.json (relative path: ./src/client)                                                                                      
    Field 'browser' doesn't contain a valid alias configuration                                      
    resolve as module                                                                                
      [path to project root]/src/client/node_modules doesn't exist or is not a directory                                                                                         
      [path to project root]/src/node_modules doesn't exist or is not a directory                                                                                                
      looking for modules in [path to project root]/node_modules                                                                                                                 
        single file module                                                                                                                                                                                
          using description file: [path to project root]/package.json (relative path: ./node_modules/@lit-labs/motion)                                                           
            no extension                                                                                                                                                                                  
              Field 'browser' doesn't contain a valid alias configuration                                                                                                                                 
              [path to project root]/node_modules/@lit-labs/motion is not a file                                                                                                 
            .tsx                                                                                     
              Field 'browser' doesn't contain a valid alias configuration                                                                                                                                 
              [path to project root]/node_modules/@lit-labs/motion.tsx doesn't exist                                                                                             
            .ts                                
              Field 'browser' doesn't contain a valid alias configuration                            
              [path to project root]/node_modules/@lit-labs/motion.ts doesn't exist                                                                                              
            .js                                                                                      
              Field 'browser' doesn't contain a valid alias configuration                            
              [path to project root]/node_modules/@lit-labs/motion.js doesn't exist                                                                                              
        existing directory [path to project root]/node_modules/@lit-labs/motion                                                                                                  
          using description file: [path to project root]/node_modules/@lit-labs/motion/package.json (relative path: .)                                                           
            using exports field: ./development/index.js                                              
              using description file: [path to project root]/node_modules/@lit-labs/motion/package.json (relative path: ./development/index.js)                                  
                no extension
                Field 'browser' doesn't contain a valid alias configuration
                  [path to project root]/node_modules/@lit-labs/motion/development/index.js doesn't exist
                .tsx
                  Field 'browser' doesn't contain a valid alias configuration
                  [path to project root]/node_modules/@lit-labs/motion/development/index.js.tsx doesn't exist
                .ts
                  Field 'browser' doesn't contain a valid alias configuration
                  [path to project root]/node_modules/@lit-labs/motion/development/index.js.ts doesn't exist
                .js
                  Field 'browser' doesn't contain a valid alias configuration
                  [path to project root]/node_modules/@lit-labs/motion/development/index.js.js doesn't exist
                as directory
                  [path to project root]/node_modules/@lit-labs/motion/development/index.js doesn't exist
```

When I inspect the `package.json` file inside `node_modules/@lit-labs/motion`, I see the following `exports` block:

```
  "exports": {
    ".": {
      "development": "./development/index.js",
      "default": "./index.js"
    }
  }
```

However, there is no `development` folder in the distribution of `@lit-labs/motion`, and this is what webpack seems to be complaining of. If I delete the `development` export in that `package.json`, my project builds correctly.

For comparison, a sister package, `@lit/reactive-element`, is distributed with a `development` folder.

--- a/packages/labs/motion/CHANGELOG.md
+++ b/packages/labs/motion/CHANGELOG.md
@@ -17,6 +17,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 <!-- ### Removed -->
 <!-- ### Fixed -->
 
+## Unreleased
+
+### Fixed
+
+- Included `development` folder in release [#1912](https://github.com/lit/lit/issues/1912).
+
 ## [1.0.0-rc.2] - 2021-04-07
 
 ### Fixed

--- a/packages/labs/motion/package.json
+++ b/packages/labs/motion/package.json
@@ -23,6 +23,7 @@
     }
   },
   "files": [
+    "/development/",
     "/src/",
     "!/src/test/",
     "/index.{d.ts,d.ts.map,js,js.map}",
@@ -33,10 +34,10 @@
   "scripts": {
     "start": "web-dev-server -- --node-resolve --watch --preserve-sym-links --open demo/index.html",
     "build": "npm run clean && tsc --build && rollup -c",
+    "build:ts": "tsc --build && treemirror development . '**/*.d.ts{,.map}'",
+    "build:ts:watch": "tsc  --build --watch",
     "clean": "rm -rf {index,flip,position}.{js,js.map,d.ts} development/ test/ *.tsbuildinfo",
     "dev": "scripts/dev.sh",
-    "build:ts": "tsc",
-    "build:ts:watch": "tsc --watch",
     "test": "npm run test:dev && npm run test:prod",
     "test:dev": "cd ../../tests && npx wtr '../labs/motion/development/**/*_test.js'",
     "test:prod": "TEST_PROD_BUILD=true npm run test:dev",

--- a/packages/labs/react/CHANGELOG.md
+++ b/packages/labs/react/CHANGELOG.md
@@ -17,6 +17,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 <!-- ### Removed -->
 <!-- ### Fixed -->
 
+## Unreleased
+
+### Fixed
+
+- Included `development` folder in release [#1912](https://github.com/lit/lit/issues/1912).
+
 ## [1.0.0-rc.2] - 2021-05-07
 
 ### Fixed

--- a/packages/labs/react/package.json
+++ b/packages/labs/react/package.json
@@ -23,6 +23,7 @@
     }
   },
   "files": [
+    "/development/",
     "/src/",
     "!/src/test/",
     "/index.{d.ts,d.ts.map,js,js.map}",

--- a/packages/labs/ssr-client/CHANGELOG.md
+++ b/packages/labs/ssr-client/CHANGELOG.md
@@ -17,6 +17,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 <!-- ### Removed -->
 <!-- ### Fixed -->
 
+## Unreleased
+
+### Fixed
+
+- Included `development` folder in release [#1912](https://github.com/lit/lit/issues/1912).
+
 ## 1.0.0-rc.1 - 2021-04-20
 
 ### Changed

--- a/packages/labs/ssr-client/package.json
+++ b/packages/labs/ssr-client/package.json
@@ -28,6 +28,7 @@
   },
   "files": [
     "/src/",
+    "/development/",
     "!/src/test/",
     "/index.{d.ts,d.ts.map,js,js.map}",
     "/directives/"

--- a/packages/labs/task/CHANGELOG.md
+++ b/packages/labs/task/CHANGELOG.md
@@ -17,6 +17,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 <!-- ### Removed -->
 <!-- ### Fixed -->
 
+## Unreleased
+
+### Fixed
+
+- Included `development` folder in release [#1912](https://github.com/lit/lit/issues/1912).
+
 ## 1.0.0-rc.2 - 2021-05-07
 
 ### Fixed

--- a/packages/labs/task/package.json
+++ b/packages/labs/task/package.json
@@ -27,6 +27,7 @@
     }
   },
   "files": [
+    "/development/",
     "/src/",
     "!/src/test/",
     "/index.{d.ts,d.ts.map,js,js.map}",

